backend/LexBoxApi/Models/Project/CreateProjectInput.cs

```diff
@@ -11,5 +11,6 @@ public record CreateProjectInput(
     string Code,
     ProjectType Type,
     RetentionPolicy RetentionPolicy,
+    bool IsSensitive,
```

Comment:
I'd like to use the same term ("confidential") in the code and the DB as in the UI. Can we change this to IsConfidential, and the rest of it accordingly (SetProjectSensitivity -> SetProjectConfidentiality and so on)? I'm worried that if we leave this named IsSensitive, we will be confused a year from now when we say "Wait, what was IsSensitive again? Oh yeah, that's "is confidential" in the UI." To be kind to our future selves, I think it might be good to rename this now before renaming it requires another DB migration.